### PR TITLE
[DOC][Kommander] Add Docker Hub Rate Limit info for Kommander.

### PR DESCRIPTION
**Pull request by russurquhart1**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

## Description of changes being made
Add Docker Hub Rate Limit info for Kommander.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/kommander/1.1/operations/manage-docker-hub-rate-limits/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<!-- markdownlint-disable MD004 MD007 MD025 MD030 MD018 MD034 -->
+
+Kommander customers can configure their cluster to authenticate with registries (such as Docker Hub), and add registries, by defining each in the `ClusterConfiguration` `.spec.imageRegistries` list in the `cluster.yaml` file.
+
+For Kommander, to add credentials for Docker Hub, set the options in your `cluster.yaml` as follows:
+
+```yaml
+- name: kommander
+  enabled: true
+  values: |2
```
Comment (cneth) [inline pages/dkp/kommander/1.1/operations/manage-docker-hub-rate-limits/index.md:20]:
```suggestion
  values: |
```

File: pages/dkp/kommander/1.2/operations/manage-docker-hub-rate-limits/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+<!-- markdownlint-disable MD004 MD007 MD025 MD030 MD018 MD034 -->
+
+Kommander customers can configure their cluster to authenticate with registries (such as Docker Hub), and add registries, by defining each in the `ClusterConfiguration` `.spec.imageRegistries` list in the `cluster.yaml` file.
+
+For Kommander, to add credentials for Docker Hub, set the options in your `cluster.yaml` as follows:
+
+```yaml
+- name: kommander
+  enabled: true
+  values: |2
```
Comment (cneth) [inline pages/dkp/kommander/1.2/operations/manage-docker-hub-rate-limits/index.md:20]:
```suggestion
  values: |
```

File: pages/dkp/kommander/1.3/operations/manage-docker-hub-rate-limits/index.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+<!-- markdownlint-disable MD004 MD007 MD025 MD030 MD018 MD034 -->
+
+Kommander customers can configure their cluster to authenticate with registries (such as Docker Hub), and add registries, by defining each in the `ClusterConfiguration` `.spec.imageRegistries` list in the `cluster.yaml` file.
+
+For Kommander, to add credentials for Docker Hub, set the options in your `cluster.yaml` as follows:
+
+```yaml
+- name: kommander
+  enabled: true
+  values: |2
```
Comment (cneth) [inline pages/dkp/kommander/1.3/operations/manage-docker-hub-rate-limits/index.md:20]:
```suggestion
  values: |
```

**Review by cneth — CHANGES_REQUESTED** [review]

**Comment by cmastr** [comment]
@russurquhart1 I think this info needs to go into the Release Notes like we did for Konvoy as well. 

https://docs.d2iq.com/dkp/konvoy/1.5/release-notes/#important-changes

File: pages/dkp/kommander/1.1/operations/manage-docker-hub-rate-limits/index.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+          [plugins."io.containerd.grpc.v1.cri".registry.configs]
+            [plugins."io.containerd.grpc.v1.cri".registry.configs."registry-1.docker.io".auth]
+                username = "<username>"
+                password = "<password>"
+                auth = ""
+                identitytoken = ""
+    ...
+    ```
+
+For more information on configuring `imageRegistries` in the `cluster.yaml`, please refer to the following documentation: https://docs.d2iq.com/dkp/konvoy/1.6/reference/cluster-configuration/v1beta2/
```
Comment (cmastr) [inline pages/dkp/kommander/1.1/operations/manage-docker-hub-rate-limits/index.md:55]: Do we need this link?

File: pages/dkp/kommander/1.2/operations/manage-docker-hub-rate-limits/index.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+title: Manage Docker Hub rate limits
+menuWeight: 25
+excerpt: Prevent issues from imposed rate limits
+beta: false
+enterprise: false
+---
+
+<!-- markdownlint-disable MD004 MD007 MD025 MD030 MD018 MD034 -->
+
+Kommander customers can configure their cluster to authenticate with registries (such as Docker Hub), and add registries, by defining each in the `ClusterConfiguration` `.spec.imageRegistries` list in the `cluster.yaml` file.
```
Comment (cmastr) [inline pages/dkp/kommander/1.2/operations/manage-docker-hub-rate-limits/index.md:13]: This should be "you" not "customers"

File: pages/dkp/kommander/1.1/operations/manage-docker-hub-rate-limits/index.md (hunk 4 above)
Comment (russurquhart1) [reply pages/dkp/kommander/1.1/operations/manage-docker-hub-rate-limits/index.md:55]: I've left it, as we got pinged for not having the same info in Kommander as Konvoy. I'm not married to it, and can take it out!

File: pages/dkp/kommander/1.3/release-notes/index.md [hunk 6]
```diff
@@ -19,6 +19,13 @@ To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [ins
 
 <p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
 
+# Important changes
```
Comment (cmastr) [inline pages/dkp/kommander/1.3/release-notes/index.md]: @russurquhart1  can we put this in 1.2 as well?

Comment (russurquhart1) [reply pages/dkp/kommander/1.3/release-notes/index.md]: Here is the JIRA for this:

https://jira.d2iq.com/browse/D2IQ-73609

**Review by cneth — CHANGES_REQUESTED** [review]
Kommander 1.3 isn't soup yet, is it?   I didn't think we even had a beta yet?  So I don't think we should have any 1.3 pages yet????

File: pages/dkp/kommander/1.3/release-notes/index.md (hunk 6 above)
Comment (russurquhart1) [reply pages/dkp/kommander/1.3/release-notes/index.md]: Put in 1.2 Release Notes.

**Review by cneth — APPROVED** [review]
Looks good, Casie filled me in that the 1.3 pages won't publish yet.

**Review by cmastr — APPROVED** [review]

**Comment by russurquhart1** [comment]
Here is the associated JIRA:

https://jira.d2iq.com/browse/D2IQ-73609

**Review by armandgrillet — APPROVED** [review]

**Pull request merged**